packages/data-point/lib/entity-types/entity-hash/resolve.test.js

```diff
@@ -101,7 +101,7 @@ describe('entity.hash.omitKeys', () => {
       })
     })
   })
-  test('do nothing if mapKeys is empty', () => {
+  test('do nothing if mapKeys is empty | hash d2', () => {
```

Comment:
Can we add `It should` to the beginning of this description? I don't think we need the stuff from the `|` on.